docs/src/main/asciidoc/logging.adoc

```diff
@@ -3,30 +3,35 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Configuring Logging
+= Configuration guidance for logging in Quarkus
```

Comment:
Hello everybody. Thank you so much for your suggestions and review provided. Before we can merge this, I suggest making a small vote on a better name and abstract so that we will have something you like while being ok with some new heading rules we have had in the Quarkus repo for some time now.


Here are my candidates for the heading.
Since this whole guide is a reference, we need to choose a heading like these:

= Configuration guidance for logging
= Logging configuration methods
= Logging configuration parameters
= Logging configuration


We cant use `Configure`, or `How to configure`, because these would be headings for a Tutorial or HowTo template, not reference. We also can use the word `reference` in the title.

For the abstract:

Here is something that should summarize it well, and yes, it also contains some portal title page "fluff". Why is it there..for searchability (google search results), we need to use some Keywords that help us get needed docs on the radar. I am also for a minimalistic approach.

Now, why we can't use something like this:
"Learn how to perform logging in your Quarkus application and configure logging output."
It is because this is not a guide that should teach you anything.

Here are some ideas. We can also create a combination.
1)
Leverage the available information on logging in Quarkus to configure it according to your specific requirements effectively.

2) Use the available information on logging in Quarkus to configure it efficiently according to your specific requirements.

3)
Use the available information on logging in Quarkus and configuring the logging output accordingly.

4)
Configure logging in your Quarkus applications to determine how data and messages about events are recorded and made available. Use the available information to set the appropriate configuration properties that you need.

Can I have your 2cents on this too?
@dmlloyd @Ladicek @mkouba @michelle-purcell

Reply:
The guide explains:

1. using logging API(s)
2. logger configuration

I'd just call it "Logging" and be done with it. It is the ultimate logging reference guide after all.

When it comes to the summary, I'll try to distill the absolute gist of it:

> Use the logging API in Quarkus efficiently. Configure logging output according to your needs. Use logging adapters to unify the output from other logging APIs.

I think those are the main topics covered in this reference.

I passionately dislike padding words like "Use available information" (or even "utilize" or "leverage"!), so I'm trying to avoid them myself :-)